x-pack/filebeat/module/squid/log/test/access2.log-expected.json

```diff
@@ -0,0 +1,5610 @@
+[
+    {
+        "@timestamp": "2002-10-23T10:25:29.000Z",
+        "event.action": "TCP_MISS",
+        "event.code": "GET",
+        "event.dataset": "squid.log",
+        "event.module": "squid",
+        "event.original": "1035368729.430    371 210.8.79.228 TCP_MISS/200 2136 GET http://www.fas.harvard.edu/~hpcws/nav/navbar_r3_c6.jpg - FIRST_PARENT_MISS/proxy2.syd.connect.com.au image/jpeg",
+        "fileset.name": "log",
+        "http.response.body.content": "navbar_r3_c6.jpg",
+        "input.type": "log",
+        "log.offset": 0,
+        "observer.product": "Proxy",
+        "observer.type": "Proxies",
+        "observer.vendor": "Squid",
+        "related.ip": [
+            "210.8.79.228"
+        ],
+        "related.user": [
+            "-"
+        ],
+        "rsa.internal.hcode": "FIRST_PARENT_MISS",
+        "rsa.internal.messageid": "GET",
+        "rsa.investigations.ec_activity": "Request",
+        "rsa.investigations.ec_subject": "NetworkComm",
+        "rsa.investigations.ec_theme": "ALM",
+        "rsa.misc.action": [
+            "GET",
+            "TCP_MISS"
+        ],
+        "rsa.misc.content_type": "image/jpeg",
+        "rsa.misc.result_code": "200",
+        "rsa.network.domain": "www.fas.harvard.edu",
+        "rsa.time.duration_time": 371,
+        "rsa.time.event_time": "2002-10-23T10:25:29.000Z",
+        "rsa.time.event_time_str": "1035368729",
+        "rsa.web.alias_host": "www.fas.harvard.edu",
+        "server.domain": "www.fas.harvard.edu",
+        "service.type": "squid",
+        "source.as.number": 2764,
+        "source.as.organization.name": "AAPT Limited",
+        "source.bytes": 2136,
+        "source.geo.continent_name": "Oceania",
+        "source.geo.country_iso_code": "AU",
+        "source.geo.location.lat": -33.494,
+        "source.geo.location.lon": 143.2104,
+        "source.ip": [
+            "210.8.79.228"
+        ],
+        "tags": [
+            "squid.log",
+            "forwarded"
+        ],
+        "url.domain": "www.fas.harvard.edu",
+        "url.original": "http://www.fas.harvard.edu/~hpcws/nav/navbar_r3_c6.jpg",
+        "user.name": "-"
+    },
+    {
+        "@timestamp": "2002-10-23T10:25:30.000Z",
+        "event.action": "TCP_MISS",
+        "event.code": "GET",
+        "event.dataset": "squid.log",
+        "event.module": "squid",
+        "event.original": "1035368730.746    297 210.8.79.228 TCP_MISS/200 1467 GET http://www.fas.harvard.edu/~hpcws/nav/navbar_r4_c1.jpg - FIRST_PARENT_MISS/proxy1.syd.connect.com.au image/jpeg",
+        "fileset.name": "log",
+        "http.response.body.content": "navbar_r4_c1.jpg",
```

Comment:
thinking we might want to just drop this field since it's not aligning with ECS

Reply:
This is using the field `webpage` which is defined as "captures the web page information". All the parsers that I've seen using it seem to do it to capture the name of the html document, so I think for now it's safe to map it to `file.name`, which is what we've done in other similar modules in the past.

WDYT?

Reply:
I think I'd be fine with `file.name`, but I'm not entirely sure if the semantics of that are fine given:

1. this doesn't necessarily have the same information as normally associated with a filesystem entry, and
2. I'm assuming that in the case of url without a file prefix this might return something like `19713` from a url like `https://github.com/elastic/beats/pull/19713`?

That said, if @ebeahan or @webmat are fine with this being `file.name` I'm fine with that too, but I'd defer to them.

Reply:
I'm fine with `file.name` in this case, but it does appear we will run into instances of the caveat in the second bullet: https://github.com/elastic/beats/blob/4f80f24fa8b6b8631e54e8e681a7cff6d78b74cb/x-pack/filebeat/module/squid/log/test/access2.log-expected.json#L1800